Script/PRONTO.py

```diff
@@ -641,15 +712,22 @@ def insert_image_to_ppt(DNA_sampleID,DNA_normal_sampleID,RNA_sampleID,DNA_image_
 	ppt.save(output_ppt_file)
 
 
-def insert_table_to_ppt(table_data_file,slide_n,table_name,left_h,top_h,width_h,left_t,top_t,width_t,height_t,font_size,table_header,output_ppt_file,if_print_rowNo):
+def insert_table_to_ppt(table_data_file,slide_n,table_name,left_h,top_h,width_h,left_t,top_t,width_t,height_t,font_size,table_header,table_header_aliase,output_ppt_file,if_print_rowNo,table_column_width):
 	table_file = open(table_data_file)
 	lines = table_file.readlines()
 	if not lines:
 		return
 	first_line = lines[0]
 	rows = len(lines)
 	first_line_cells = first_line.split('\t')
-	cols = len(first_line_cells) -1
+	cols = len(table_header)
+	header_not_exist_InTable = []
+	for n in range(len(table_header)):
```

Comment:
also a good candidate for its own function:

```suggestion
	header_not_exist_in_table = identify_missing_headers(table_header, first_line_cells)
```